Filter/DateRangeFilter.php

```diff
@@ -11,19 +11,15 @@
 
 namespace Sonata\AdminSearchBundle\Filter;
 
-class DateRangeFilter extends AbstractDateFilter
-{
-    /**
-     * This is a range filter.
-     *
-     * @var bool
-     */
-    protected $range = true;
+use Sonata\AdminBundle\Form\Type\Filter\DateRangeType;
 
+class DateRangeFilter extends AbstractDateFilter implements RangeFilterInterface
+{
     /**
-     * This filter has time.
-     *
-     * @var bool
+     * {@inheritdoc}
      */
-    protected $time = false;
+    protected function getFilterTypeClass()
```

Comment:
I think you could make it final

Reply:
If we are not going to allow them to override this, then we make the class final, correct?

Reply:
Yeah but making the class final is a BC-break, while making a new method final isn't

Reply:
True. But in case if these are acceptable BC then the class is final. (Still waiting for @sonata-project/contributors to say if it acceptable )